contents/blog/sql-for-analytics.md

```diff
@@ -0,0 +1,137 @@
+---
+title: The basics of SQL for analytics
+date: 2023-09-04
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Guides
+ - Explainers
+---
+
+One of SQL’s most important use cases is analytics. There are many ways to slice, modify, and interpret analytics data. SQL provides the depth and customizability to get insights from it, answer questions, and make data-driven decisions.
+
+This post goes over how to analyze your analytics data with SQL and recommendations for your analysis process.
+
+## What is SQL?
```

Comment:
```suggestion
## Why use SQL for analytics?
```

Reply:
Oops, I meant to just delete this section, I added the parts to before and after